CHANGELOG.md

```diff
@@ -1,3 +1,304 @@
+  and verify the version using `lokoctl version`:
+
+  ```bash
+  v0.9.0
+  ```
+
+##### Update steps for Equinix Metal (formerly Packet)
+
+   a. Backup the Terraform state file:
+   ```bash
+   cd $assets_dir/terraform
+
+   # If using local backend
+   cp terraform.tfstate backup.tfstate
+
+   # If using AWS S3 backend
+   terraform state pull > backup.state
+   ```
+
+   b. Update Terraform provider from `packethost/packet` to `equinix/metal`:
+   ```bash
+   terraform state replace-provider packethost/packet equinix/metal
+   ```
+
+   c. Pull the latest state file (required, only if using S3 backend):
+   ```bash
+   terraform state pull > terraform.tfstate
+   ```
+
+   d. Replace all references of `packet_` with `metal_` in the state file:
+   ```bash
+    sed -i 's/packet_/metal_/g' terraform.tfstate
+   ```
+
+   e. Change the module name from `module.packet` to `module.equinixmetal` in
+   the state file:
+   ```bash
+   sed -i 's/module.packet/module.equinixmetal/g' terraform.tfstate
+   ```
+
+   f. Replace `packet` with `equinixmetal` in the cluster configuration file:
+   ```hcl
+   # old
+   cluster "packet" {
+   ...
+   }
+
+   # new
+   cluster "equinixmetal" {
+   ...
+   }
+   ```
+
+   g. Uninstall Packet CCM as we are replacing it with Cloud Provider Equinix Metal.
+   ```bash
+   helm uninstall packet-ccm --namespace kube-system
+   ```
+
+   g. Upgrade to Lokomotive v0.9.0.
+   ```bash
+   lokoctl cluster apply --skip-components --skip-pre-update-health-check
+   ```
+
+   **NOTE**: Do not forget the `--skip-pre-update-health-check` flag.
+
+##### Update steps for Baremetal
+
+   a. Create new files in the assets directory with the file name the MAC addresses of the
+   controller and worker nodes and the contents of the file being the domain name:
+   ```bash
+   # for each controller and worker nodes
+   echo <DOMAIN_NAME> > $assets_dir/cluster-assets/<MAC_ADDRESS>
+   ```
+
+   b. Change the value of `k8s_domain_name` to only include the domain name:
+   Example:
+   ```bash
+   # old
+   k8s_domain_name = mercury.example.com
+
+   # new
+   k8s_domain_name = "example.com"
+   ```
+
+   c. Add a `pxe_commands` entry which lokoctl uses to automate the PXE (re)provisioning. For existing clusters
+   you can use `pxe_commands = "true"` to have no PXE automation (`true` is the no-op bash shell command),
+   and reprovisioning through PXE won't be supported for this cluster.
+
+   d. Follow the steps mentioned in [this section](#baremetal-features-user-data-changes-and-reprovisioning-of-worker-nodes)
+   as per the desired upgrade path. Make the necessary configuration changes as mentioned.
+   Finally execute:
+   ```bash
+   lokoctl cluster apply --skip-components
+   ```
+
+##### Other platforms
+
+   ```bash
+   lokoctl cluster apply --skip-components
+   ```
+
+#### Component update steps
+
+   Update installed Lokomotive components:
+   ```bash
+   lokoctl components apply
```

Comment:
No special steps needed for any component?

Reply:
Nopes just the configuration syntax changes for Prometheus operator, which are already mentioned in the configuration syntax changes section.